test/test-ingress.js

```diff
@@ -121,6 +123,42 @@ contract ('Ingress contract', (accounts) => {
         })
     }),
 
+    describe('Ingress contract adding same contract key updates that registry entry', () => {
```

Comment:
There is no reason to use `describe()` inside a `contract()` block
https://truffleframework.com/docs/truffle/testing/writing-tests-in-javascript#use-contract-instead-of-describe-